src/clients/java/src/main/java/com/tigerbeetle/AccountBalanceBatch.java

```diff
@@ -75,6 +75,17 @@ void setDebitsPending(final BigInteger debitsPending) {
         putUInt128(at(Struct.DebitsPending), UInt128.asBytes(debitsPending));
     }
 
+    /**
+     * @param debitsPending an array of 16 bytes representing the 128-bit value.
+     * @throws IllegalArgumentException if {@code debitsPending} is not 16 bytes long.
+     * @throws IllegalStateException if not at a {@link #isValidPosition valid position}.
+     * @throws IllegalStateException if a {@link #isReadOnly() read-only} batch.
+     * @see <a href="https://docs.tigerbeetle.com/reference/account-balances#debits_pending">debits_pending</a>
+     */
+    void setDebitsPending(final byte[] debitsPending) {
+        putUInt128(at(Struct.DebitsPending), debitsPending);
+    }
```

Comment:
I don't think we actually need these implementations, but not sure if it is worth specifically omitting them either.